IssueComment
  { issueCommentUpdatedAt = 2013 (-03) (-22) 19 : 02 : 31 UTC
  , issueCommentUser =
      SimpleUser
        { simpleUserId = Id 48436
        , simpleUserLogin = N "coffeemug"
        , simpleUserAvatarUrl =
            "https://avatars.githubusercontent.com/u/48436?v=3"
        , simpleUserUrl = "https://api.github.com/users/coffeemug"
        , simpleUserType = OwnerUser
        }
  , issueCommentUrl =
      "https://api.github.com/repos/rethinkdb/rethinkdb/issues/comments/15315502"
  , issueCommentHtmlUrl =
      "https://github.com/rethinkdb/rethinkdb/issues/525#issuecomment-15315502"
  , issueCommentCreatedAt = 2013 (-03) (-22) 19 : 02 : 31 UTC
  , issueCommentBody =
      "This is in debug mode -- the sever sends a stack trace in debug mode back to the client. In release mode you wouldn't see the failed assertion, just the compile error. That one makes sense, albeit it's very confusing. Also, the underlining (or backtrace) are wrong."
  , issueCommentId = 15315502
  }